runtime/compiler/env/j9method.cpp

```diff
@@ -3012,7 +3012,11 @@ void TR_ResolvedJ9Method::construct()
 
    static X VectorSupportMethods[] =
       {
+#if JAVA_SPEC_VERSION == 21
```

Comment:
is the jdk21 version the same as jdk17?

Reply:
> is the jdk21 version the same as jdk17?

I would need to check. So far, we only tried to accelerate one version at a time. So, without this PR, most likely only Java 21 is accelerated. With this PR, Java 21 and Java 22 will be accelerated. By acceleration, I mean partial acceleration of course, and only on some platforms, since this is still an incubator.

Reply:
Java 17 Vector API is definitely not supported right now since the implementation of the VectorAPI incubator has changed a lot since then.

Reply:
I expect we dropped jdk17 support some time ago, and no one has complained.

Reply:
Okay, I was asking because if this is compiled with jdk17 it picks up the kdk22 version. But it sounds like this doesnt matter

Reply:
its probably safer to do `#if JAVA_SPEC_VERSION <= 21` but Ill leave that up to you

Reply:
Yes, I agree. Let me change to` <= 21`